### DISCUSSION TITLE
Trouble exporting (and then importing) .db files correctly in SQLite Studio

### DISCUSSION THREAD

**Discussion by The-TurtleLord** [pawelsalawa/sqlitestudio#4463] [Q&A] [answered]
Hi everyone. 
So I have made a database in SQLite Studio, and I need to export it as a .db file.
However, after exporting the file (apparently successfully) I am then unable to open that .db file up in SQLite Studio again. Any ideas? I am able to open other .db files which I've downloaded from a data repository for my class, it is just that I exported myself which is troubling me. 
I've attached some screenshots below, let me know if you think the issue is the format i have exported it, the fact that I have the original database open in SQLite Studio still, or something else :]
Thanks heaps for the help!

![exp1](https://user-images.githubusercontent.com/5248256/173051396-2688bb7d-a899-45d2-9aee-3c1bd344194b.PNG)

![exp2](https://user-images.githubusercontent.com/5248256/173051380-eff387af-a5d8-4b5f-8d71-7e5d59972bd6.PNG)

![exp3](https://user-images.githubusercontent.com/5248256/173051416-b1e6949e-0f90-4ec3-9d27-a934039955d9.PNG)

**Comment by pawelsalawa** [comment, marked as answer]
The db file does not need to be exported in your case. SQLite database **is** the db file. 
In other words, hold your mouse over your "Vacations" database in the list of databases (left-side panel) and in the tooltip you will see the path to the database file. Just make a copy of that file. Voila! You can now add a copy of that file to SQLiteStudio as another database.

**Reply by The-TurtleLord** [reply]
Thank you so much! I think my understanding of what a db file actually is was very low haha, hence the confusion. So whenever we make a change to the database in SQLite Studio, we are updating the underlying db file? 
I will also say in my defense, the file was very well hidden in my folders haha.  :]

**Reply by pawelsalawa** [reply]
> whenever we make a change to the database in SQLite Studio, we are updating the underlying db file?

Correct.
